content/terms/declarations-maintenance.en.md

```diff
@@ -106,3 +106,11 @@ If the service provider stops offering a service, the associated terms will beco
 ## How to rename a service
 
 The consensus is to consider that a service provider renaming a service (for example, `Twitter` to `X`) is akin to terminating the previous service and opening a new one. Therefore, to apply a service renaming, open a pull request that both [terminates the previous service](#how-to-terminate-a-service) and adds a new [service declaration]({{< relref "terms/tracking-new-terms#declaring-a-new-service" >}}) with the new service name. You can reuse the `documents` part of the original declaration, but should double-check that the selectors and URLs still match, as a service rename is most often accompanied by a new page layout, a new domain name, and sometimes entirely new terms.
+
+## How to get the `validUntil` date from an issue
+
+1. Go to the open issue and position yourself on the last comment from OTA-Bot.
+2. On the date of this comment, right-click "inspect" to open your browser's code inspector. Note that sometimes, the date is not quite precise, it can be written "last month".
+![alt text]({{< relref "." >}}ota-bot-comment.png)
```

Comment:
`alt text` is a problematic entry as an alt text 😉 either remove it or actually write alt text!

Reply:
what an oversight! thank you